src/extensions/surveys/surveys-utils.ts

```diff
@@ -587,8 +587,8 @@ export const createOpenTextOrLinkPopup = (
                         [`$survey_responded/${survey.id}`]: true,
                     },
                 })
-                if (surveyQuestionType === 'link') {
-                    window.open(question.link || undefined)
```

Comment:
Curious, why did we do it like this previously? `window.open(undefined)` opens an empty tab.

Reply:
we originally required link urls for link type surveys so the button was always going to open something one way or another 😅 

I think window.open("") would open something weird so passing in undefined was better

Reply:
What does the CTA do now, if it doesn't link to something? 🤔